packages/api-auth-lib/OAuth2Manager.js

```diff
@@ -5,6 +5,12 @@ const open = require('open');
 
 const { HubSpotAuthError } = require('./Errors');
 const { handleExit } = require('@hubspot/cms-lib/lib/process');
+const {
+  getEnv,
+  getEnvForConfig,
+  getHubSpotWebsiteDomain,
+  getHubSpotApiDomain,
+} = require('@hubspot/cms-lib/lib/environment');
```

Comment:
Sorry for not noticing this when the `handleExit` change was made, but this creates a circular dependency relationship between `@hubspot/cms-lib` and `@hubspot/api-auth-lib`, which I am a bit surprised hasn't caused issues.

Reply:
As per our conversation, I added these methods to api-auth-lib to avoid any potential cyclic dep